UI/HTML/adminsignup.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="../css/style.css">
   <style>@import url('https://fonts.googleapis.com/css?family=Oxygen');
```

Comment:
#### Critical
This should go into the `<link>` tag. No need to create it inside a style tag

Reply:
@adex001 
This is still pending.

#### Critical
Use `<link>` over `@import` as much as possible.
Move this to the `<link>` tag i.e.
```html
<link rel="stylesheet" url="https://.....">
```